### ISSUE TITLE
Inline code support in the editor?

### ISSUE THREAD

**Issue by lf-** [ubc/compair#965]
Hi!

In a math course, it's often nice to be able to type LaTeX suggestions in the box, but it's very unclear whether these get rendered (sort of losing their purpose): when I typed in `$ ... $`, to my surprise, when viewing my comparisons later, it ended up rendered with math mode in MathJaX. It would be nice to be able to see how this gets rendered before submitting comparisons, and also to be able to force it to not render by putting the suggestion in inline code.

This was observed on the live UBC site.

**Comment by kitsook** [comment]
Thanks for your suggestions.  The editor itself is already in WYSIWYG mode.  The issue is with the embedded latex code.  Especially when the code fragment is pasted into the editor (i.e. no paragraph breaks between lines).  This may confuse the MathJax renderer.

When entering code, consider to use the code snippet formatter:
![image](https://user-images.githubusercontent.com/13360325/95134780-51ec9200-0718-11eb-9668-cdf9c19b8961.png)

If you cannot see the language you are using (e.g. TeX), just leave the Language selection blank.  e.g.
![image](https://user-images.githubusercontent.com/13360325/95134905-7a748c00-0718-11eb-989f-e2d913378010.png)

Result:
![image](https://user-images.githubusercontent.com/13360325/95135660-9e849d00-0719-11eb-82ec-53b6baf1f0bd.png)

Hope this helps.

**Comment by lf-** [comment]
Yeah, I found that button, but it would be good to have inline code so I can include code within my prose rather than having to make it a separate paragraph, especially if my code is `this long`, which it often is.
